activation/activation.go

```diff
@@ -677,7 +670,6 @@ func (b *Builder) createAtx(
 		nipostState.NumUnits,
 		nonce,
 	)
-	atx.InnerActivationTx.NodeID = atxNodeID
```

Comment:
The NodeID is required to be set in an initial ATX; this cannot just be dropped here?

Reply:
Correct, but it must be set in the wire type `wire.ActivationTxV1`. This field doesn't need to be present in `types.ActivationTx` and hence it was removed. It's set in https://github.com/spacemeshos/go-spacemesh/blob/1ca59a4a975ba3210f37dbdc737b153fb05a4f90/common/types/activation.go#L429-L431